src/Files.App/Helpers/ResourceHelpers.cs

```diff
@@ -6,16 +6,10 @@ namespace Files.App.Helpers
 	[MarkupExtensionReturnType(ReturnType = typeof(string))]
 	public sealed class ResourceString : MarkupExtension
 	{
-		private static ResourceLoader resourceLoader = new ResourceLoader();
+		private static readonly ResourceLoader loader = new();
```

Comment:
Readonly is alright but lets keep the names as they were before.

Reply:
It's done.